PROTOCOL.md

```diff
@@ -0,0 +1,99 @@
+# GraphQL over WebSocket Protocol
```

Comment:
rough example: :)
Client initates connection
C => `{"type": "connection_init", payload: "..." }`
Server may reject the connection:
S => `{"type": "connection_error", payload: "Permission denid"}`
or accept it with 
S => `{"type": "connection_ack"}`
etc...
like add for each packet type a json example :)